task_2/src/controllers/__tests__/groupController.test.ts

```diff
@@ -0,0 +1,71 @@
+import request from 'supertest'
+import jwt from 'jsonwebtoken'
+import { localHostUrl } from '../../utils/constants.js'
+
+const accessToken = jwt.sign(
+  {
+    login: 'testUser',
+  },
+  `${process.env.ACCESS_TOKEN_SECRET}`,
+  { expiresIn: '5m' }
+)
+const authorization = `Bearer ${accessToken}`
+
+describe('GroupController', () => {
+  it('should get all available groups', async () => {
+    const response = await request(localHostUrl).get('/group/all').set('Authorization', authorization)
+
+    expect(response.status).toEqual(200)
+    expect(response.body.data.length).toEqual(6)
```

Comment:
here keep in mind that the length will not be 6 all the time so it is better to use toBeGreaterThan(0) rather than toEqual(6)

Reply:
noted, thank you